extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -0,0 +1,257 @@
+        },
+        "https://ns.adobe.com/experience/campaign/link/id": {
+          "title": "Tracking URL ID",
+          "type": "string",
+          "description": "Unique Identifier of the Link"
+        },
+        "https://ns.adobe.com/experience/campaign/link/description": {
+          "title": "Tracking URL Description",
+          "type": "string",
+          "description": "A human-friendly description of the Link."
+        },
+        "https://ns.adobe.com/experience/campaign/link/url": {
+          "title": "URL",
+          "format": "uri",
+          "type": "string",
+          "description": "URL of the Link."
+        },
+        "https://ns.adobe.com/experience/campaign/link/occurrence": {
+          "title": "Occurrence",
+          "type": "integer",
+          "description":
+            "URL occurrence index in the content (first is 1). When the same URL is available several times in the same content, its is necessary to track which occurrence of the URL has been clicked on.",
+          "default": 1,
+          "minimum": 1,
+          "maximum": 32767
+        },
+        "https://ns.adobe.com/experience/campaign/link/tags": {
+          "title": "Tags",
+          "type": "array",
+          "description":
+            "A list of tags qualifying the link. The URL category is one of those tags.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "https://ns.adobe.com/experience/campaign/message/offerProposed": {
+          "title": "Offers",
+          "type": "array",
+          "description": "Offers displayed in the message.",
+          "items": {
+            "$ref":
+              "https://ns.adobe.com/xdm-extensions/adobe/experience/campaign/offer"
+          }
+        },
+        "https://ns.adobe.com/experience/campaign/message/offerClicked": {
+          "title": "Offer Clicked",
+          "$ref":
+            "https://ns.adobe.com/xdm-extensions/adobe/experience/campaign/offer",
+          "description":
+            "The offer and its location in the message for a click event related to an offer."
+        },
+        "https://ns.adobe.com/experience/campaign/message/externalId": {
+          "title": "External ID",
+          "type": "string",
+          "description":
+            "An ID generated by the parent activity invoking service to check later if the incoming experienceEvent was generated using this ID."
+        },
+        "https://ns.adobe.com/experience/campaign/businessContext": {
+          "title": "Business Context",
+          "type": "string",
+          "description":
+            "Internal field for identifying the various actions/intents."
```

Comment:
This is almost worse than useless. What are the values that this can take?

Reply:
the original request comes from the internal issue plat-14497

Reply:
In assets, we have a property `layerStyles` that controls how the layer is styled. All that is known about the property is that it is a `string`.

This property clearly is a structured property that either points to another value or encodes the actions or intents. If that's the case, we should either document it, or leave it out entirely.